lib/que/active_support/job_middleware.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Que
+  module ActiveSupport
+    module JobMiddleware
+      def self.call(job)
+        labels = {
+          job_class: job.que_attrs[:job_class],
+          priority: job.que_attrs[:priority],
+          queue: job.que_attrs[:queue],
+          latency: job.que_attrs[:latency],
```

Comment:
I think it would be also useful to have a `duration` published, so we know both how long the job was waiting in the queue, and how long it took to execute. WDYT?

Reply:
Execution duration can be calculated using start and end currently, we could make this more intuitive by doing that for consumers